chore(qbitrr): database files no longer in app setup

Feramance opened the pull request:
**Description**
Database files no longer required as of v4
⚒️ Fixes  # <!--(issue)-->

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
No testing required

**📃 Notes:**
None

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [x] 📄 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

Feramance committed:
Update installation-notes.md

database files no longer requried as of v4

Signed-off-by: Feramance <38938175+Feramance@users.noreply.github.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/truecharts/charts?pullRequest=17071) <br/>All committers have signed the CLA.

Feramance committed:
Update installation-notes.md

Added more notes specifying the flexibility of the number of managed Sonarr/Radarr instances

Signed-off-by: Feramance <38938175+Feramance@users.noreply.github.com>

Crow-Control commented:
Please use correct titles for PR's next time.
